src/Microsoft.AspNetCore.Mvc.ViewFeatures/project.json

```diff
@@ -47,18 +47,15 @@
       "type": "build"
     },
     "Microsoft.Extensions.WebEncoders": "1.0.0-*",
-    "Newtonsoft.Json": "8.0.3",
+    "Newtonsoft.Json": "8.0.4-beta1",
```

Comment:
I'll release a stable version that supports netstandard when .NET Core is stable so you'll be waiting a while.